### ISSUE TITLE
WARNING: Contrast ratio of #fff on #ffc82c is pretty bad

### ISSUE THREAD

**Issue by pranavrajs** [chatwoot/chatwoot#117]
```
16:42:10 frontend.1 | WARNING: Contrast ratio of #fff on #ffc82c is pretty bad, just 1.5
16:42:10 frontend.1 |          on line 75 of node_modules/foundation-sites/scss/util/_color.scss, in function `color-pick-contrast`
16:42:10 frontend.1 |          from line 121 of node_modules/foundation-sites/scss/components/_button.scss, in mixin `button-style`
16:42:10 frontend.1 |          from line 236 of node_modules/foundation-sites/scss/components/_button.scss, in mixin `foundation-button`
16:42:10 frontend.1 |          from line 81 of node_modules/foundation-sites/scss/foundation.scss, in mixin `foundation-everything`
16:42:10 frontend.1 |          from line 3 of node_modules/foundation-sites/assets/foundation-flex.scss
16:42:10 frontend.1 |          from line 8 of app/javascript/src/assets/scss/app.scss
16:42:10 frontend.1 |          from line 28 of stdin
```

**Comment by shreve** [comment]
Foundation wants the contrast to be 3 or greater. There are two ways to fix this:

1. Pick a warning color with a higher luminance contrast against white.
2. Disable contrast warnings in foundation.

If you prefer the latter, I can handle that.

**Comment by pranavrajs** [comment]
@shreve Thanks for taking the time to look at the issue. I would prefer to go with solution 1 rather than disabling all contrast warnings. Please feel free to assign this to yourself before you plan to work on it.

**Comment by nithindavid** [comment]
@shreve Let us know if you are taking up this issue :)

**Comment by AndreiUngur** [comment]
Should be solved by https://github.com/chatwoot/chatwoot/pull/144

**Comment by sojan-official** [comment]
resolved in #144
